examples/AdvancedOperations/GetAdGroupBidModifiers.php

```diff
@@ -100,7 +100,7 @@ public static function main()
     public static function runExample(
         GoogleAdsClient $googleAdsClient,
         int $customerId,
-        int $adGroupId
+        ?int $adGroupId
```

Comment:
Comments are not updated in consequence: here it should be something like "int|null"